pkgdb/src/resolver/mixins.cc

```diff
@@ -288,9 +288,9 @@ EnvironmentMixin::addManifestFileArg( argparse::ArgumentParser & parser,
                                       bool                       required )
 {
   argparse::Argument & arg
-    = parser.add_argument( "manifest" )
+    = parser.add_argument( "--manifest" )
```

Comment:
I wasn't sure if this was the best way to make this change. I think most of the `Environment` class is not written with an optional manifest in mind. But it seems to work, and it seems like `search` uses an empty manifest just fine. Not sure if there's a better approach now, or maybe we can just clean up in the future

Reply:
Yeah we should probably make that member optional in another PR soon. It's been kind of wonky having to pass an empty manifest manually.

This works for now though.